clang/lib/Sema/SemaX86.cpp

```diff
@@ -45,6 +45,14 @@ bool SemaX86::CheckBuiltinRoundingOrSAE(unsigned BuiltinID, CallExpr *TheCall) {
   case X86::BI__builtin_ia32_vcvttsh2si64:
```

Comment:
@phoebewang These don't have sis32/64 either.
I'm not sure what is the convention here.

Reply:
We don't have a VCVTTSH2SIS instruction. The general convention is to use the instruction name directly.